.github/workflows/test.yml

```
@@ -69,13 +69,10 @@ jobs:
       fail-fast: false
       matrix:
         python-version: ["3.9", "3.10", "3.11"]
-        hoomd-version: ["2.9.7", "3.11.0", "4.7.0"]
+        hoomd-version: ["3.11.0", "4.9.1"]
         include:
           - python-version: "3.12"
             hoomd-version: "4.7.0"
```

This should have been bumped as well:
```suggestion
            hoomd-version: "4.9.1"
```